### ISSUE TITLE
endpoints doc rendered incorrectly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7538]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7538|
|Type|enhancement|
|Created|2014-07-12 04:41:06Z|
|Branch|https://github.com/twisted/twisted/tree/endpoints-docs-7538|

That's because there's a missing space.

<details><summary>Searchable metadata</summary>

```
trac-id__7538 7538
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_endpoints_docs_7538 branches/endpoints-docs-7538
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1405140066536256 1405140066536256
changetime__1405415155099566 1405415155099566
version__None None
owner__Alex Alex

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

Fixed by https://github.com/twisted/twisted/pull/33

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to Alex|
|-|-|

Looks fine (apart from missing misc news fragment), please merge.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Alex commented|
|-|-|

Hi exarkun. I'm not a committer on twisted, someone else needs to merge this :-) I'll push a `.misc` though.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh commented|
|-|-|

(In [42869]) Branching to endpoints-docs-7538.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh set status to `closed`|
|-|-|

(In [42871]) Merge endpoints-docs-7538: fix a ReST markup issue in endpoint docs

Author: Alex
Reviewer: exarkun
Fixes: [#7538](https://github.com/twisted/twisted/issues/7538)

Due to a ReST markup issue in the endpoint documentation, there
were rendering issues in the resulting HTML. This adds the missing
space.
